packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External Dependencies
+ */
+import classnames from 'classnames';
+/**
+ * Internal Dependencies
+ */
+import type { Config, TourStepRendererProps } from '../types';
+
+interface Props extends TourStepRendererProps {
+	config: Config;
+}
+
+const TourKitStep: React.FunctionComponent< Props > = ( {
+	config,
+	steps,
+	currentStepIndex,
+	onMinimize,
+	onDismiss,
+	onNext,
+	onPrevious,
+	setInitialFocusedElement,
+	onGoToStep,
+} ) => {
+	const classNames = classnames(
+		'tour-kit-step',
+		`is-step-${ currentStepIndex }`,
+		config.options?.className ? `${ config.options?.className }__step` : '',
+		config.steps[ currentStepIndex ].className
```

Comment:
Kinda think this could be nested under `config.steps[ currentStepIndex ].options` for consistency, like the parent one, but maybe rethink later.